sf/draft procurement/stmt of qual.md

```diff
@@ -31,22 +31,26 @@ Play 2 – Address the Whole Experience from Start to Finish
   
 Play 3 – Make it Simple and Intuitive
   * The products or services for our customers/clients do not require knowledge of the organizational structure or our agency to complete the process.
-  * We coordinated with our customers/clients to make design and content decisions based on user research, usability testing, and analytics.
-  * For all user-facing features built over the last quarter, we coordinated with our customers/clients to use plain language that is familiar to the user and easy to understand.
+  * We coordinated with our customers/clients to make design and content decisions based on accessibility needs, user research, usability testing, and analytics.
```

Comment:
- We added a note on accessibility specifically to ensure compliance and inclusiveness